app/api/services/briefs.py

```diff
@@ -28,3 +30,38 @@ def get_supplier_responses(self, code):
             .all()
 
         return [r._asdict() for r in responses]
+
+    def get_user_briefs(self, current_user_id):
+        """Returns summary of a user's briefs with the total number of sellers that applied."""
+        results = (db.session.query(Brief.id, Brief.data['title'].astext.label('name'),
+                                    Brief.closed_at, Brief.status, func.count(BriefResponse.id).label('applications'),
+                                    Framework.slug.label('framework'), Lot.slug.label('lot'))
+                   .join(BriefUser, Framework, Lot)
+                   .filter(current_user_id == BriefUser.user_id)
+                   .outerjoin(BriefResponse, Brief.id == BriefResponse.brief_id)
+                   .group_by(Brief.id, Framework.slug, Lot.slug)
+                   .order_by(sql_case([
+                       (Brief.status == 'draft', 1),
+                       (Brief.status == 'live', 2),
+                       (Brief.status == 'closed', 3)]), Brief.closed_at.desc().nullslast())
+                   .all())
+
+        return [r._asdict() for r in results]
+
+    def get_team_briefs(self, current_user_id, domain):
+        """Returns summary of live and closed briefs submitted by the user's team."""
+        team_ids = db.session.query(User.id).filter(User.id != current_user_id,
+                                                    User.email_address.endswith(concat('@', domain)))
+
+        team_brief_ids = db.session.query(BriefUser.brief_id).filter(BriefUser.user_id.in_(team_ids))
+
+        results = (db.session.query(Brief.id, Brief.data['title'].astext.label('name'), Brief.closed_at, Brief.status,
+                                    Framework.slug.label('framework'), Lot.slug.label('lot'), User.name.label('author'))
```

Comment:
Should team briefs include the same brief 'applications' count like what is provided in the user briefs above? Thinking from a front-end perspective, I think 'My briefs' and 'Team briefs' should be presented the same where possible.

Reply:
Design team have indicated this is not a priority at the moment.